app/controllers/tag_controller.rb

```diff
@@ -132,8 +132,14 @@ def show
     end
     nodes = nodes.where(created: @start.to_i..@end.to_i) if @start && @end
 
-    @notes = nodes.where('node.nid NOT IN (?)', qids) if @node_type == 'note'
-    @questions = nodes.where('node.nid IN (?)', qids) if @node_type == 'questions'
+    if qids.empty?
+      @notes = nodes
+      @questions = nodes
```

Comment:
Here, if there are no questions, could we just assign `[]` to `@questions`? We wouldn't want to pass in non-question nodes just because there are no questions, perhaps?

Reply:
Yeah, I guess you're right 👍